training/run_distillation.py

```diff
@@ -1674,6 +1667,11 @@ def generate_step(batch):
 
                 # break condition
                 if cur_step == total_train_steps:
+
+                    # un-wrap student model for save
+                    student_model = accelerator.unwrap_model(student_model)
+                    student_model.save_pretrained(training_args.output_dir)
```

Comment:
We just need to remember to push to hub after this last save step

Reply:
Resolved in https://github.com/huggingface/distil-whisper/pull/117/commits/bcf7a5274dede8dbf9d30ed0a32caf9303dd44b7